lib/BaseController.js

```diff
@@ -543,8 +543,19 @@ module.exports = class BaseController {
     if (liveResource) {
       let debug = objectPath.get(liveResource, ['metadata', 'labels', 'deploy.razee.io/debug']) ||
         objectPath.get(liveResource, ['metadata', 'labels', 'kapitan.razee.io/debug'], 'false');
+      if (objectPath.get(file, ['metadata', 'annotations']) === null) {
+        objectPath.set(file, ['metadata', 'annotations'], {});
+      }
```

Comment:
this isnt entirely necessary, but it doesnt really hurt to make sure annotations is not null before trying to write to it